build.gradle

```diff
@@ -22,7 +22,7 @@ plugins {
     id "com.github.johnrengelman.shadow" version "7.0.0"
 }
 
-def versionInfo = [major: '1', minor: System.getenv("BUILD_NUMBER").toString()]
+def versionInfo = [major: '2', minor: System.getenv("BUILD_NUMBER") ?: "0"]
```

Comment:
This defaults to build 0 so building locally doesn't break

Reply:
I always put this as "DEV" so to not interfere with production builds.

Reply:
Since this build would rely on an alpha of JDA, wouldn't it also make sense to like manually add a `alpha` or similar to the version here?

Reply:
I'll likely just release `2.0-SNAPSHOT`s until a full release.

Reply:
Added `-SNAPSHOT`